public/app/features/explore/Logs/Logs.test.tsx

```diff
@@ -243,6 +271,49 @@ describe('Logs', () => {
 
       expect(scrollElementMock.scroll).not.toHaveBeenCalled();
     });
+
+    it('should call reportInteraction on permalinkClick', async () => {
+      Object.defineProperty(window, 'location', {
```

Comment:
I might be wrong in 2023, but I think you need to restore these modifications in an `afterAll`.